nat-lab/tests/conftest.py

```diff
@@ -38,3 +40,27 @@ def event_loop():
         finally:
             asyncio.events.set_event_loop(None)
             loop.close()
+
+
+def idfn(val):
```

Comment:
Can't this be applied for all tests during a collection or using this `pytest_make_parametrize_id` hook, instead of manually importing and assigning this function to every parameter id? Just an idea, not 100% sure if possible

Reply:
Oh, if returning `None` uses the default mangling logic, then there should be no issues.

Reply:
Added hook